graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/DgsExecutionResult.kt

```diff
@@ -14,86 +14,39 @@
  * limitations under the License.
  */
 
-package com.netflix.graphql.dgs.internal
+package com.netflix.graphql.dgs
 
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.databind.exc.InvalidDefinitionException
 import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
 import com.netflix.graphql.dgs.internal.utils.TimeTracer
 import graphql.ExecutionResult
 import graphql.ExecutionResultImpl
-import graphql.GraphQLError
 import graphql.GraphqlErrorBuilder
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 import org.springframework.http.HttpHeaders
 import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
 
-class DgsExecutionResult @JvmOverloads constructor(
+class DgsExecutionResult(
     val executionResult: ExecutionResult,
```

Comment:
I wonder if this property needs to be public, we are already delegating all methods to it